### ISSUE TITLE
[1.7.10] Furnace, Forge Multipart and vanishing item bugs

### ISSUE THREAD

**Issue by Chadramar** [Dynious/RefinedRelocation#215]
RefinedRelocation-1.7.10-1.0.6b
Forge 10.13.0.1208

When extracting non-oak wood from a chest to a furnace, the remainder of the stack will turn into oak wood as soon as the first piece (which will remain its original type) is on the way. Similarly, charcoal will turn into coal. I also couldn't get the extraction module to work on the furnace itself, though connecting a block extender to the furnace and extracting it through that works.

When adding ForgeMultipart (1.7.10-1.1.0.300) to an existing world, I get an error message about RefinedRelocation:relocator being missing. Loading the save anyway turned the relocators into power limiters. Also, loading a save with FMP active causes modules attached to block extenders to pop off.

All verified on a clean profile in its own folder with no other mods installed.

With or without FMP, I also notice items and entire stacks vanishing when an extraction module sent several stacks to say a furnace at the same time. Some bounce back to the source, and if this repeats it eventually leaves behind a single pseudo-item that looks like and is called the same as say normal coal, but does not stack with other coal: it removes the stack and leaves only itself behind. It will also not be extracted to the furnace. Put in the furnace as fuel, this coal does work and in fact caused a new piece of coal to appear in the fuel slot once the first burned out. This happened several times. I then removed that pseudo-stack from the furnace, saved and reloaded, and tried again to combine it with an existing normal stack of coal. This caused the original 19 coal to turn into 6 (which burned normally in the furnace).

On the other hand, a pseudo-ore created by rebounding will be smelted only once (it too had to be inserted manually, the extraction module did not work). Combined with a stack, it turned into a smaller stack which smelted down normally, instead of a pseudo-stack.

This does not seem to happen when the destination inventory is a chest -- the items that don't find space just bounce back to the source chest "intact".

**Comment by Dynious** [comment]
2 is duplicate of #145 rest was fixed a long time ago.
